SocialCareCaseViewerApi/V1/Controllers/RelationshipController.cs

```diff
@@ -79,5 +79,33 @@ public IActionResult CreatePersonalRelationship([FromBody] CreatePersonalRelatio
                 return BadRequest(e.Message);
             }
         }
+
+        /// <summary>
+        /// Delete a personal relationship
+        /// </summary>
+        /// <param name="id"></param>
+        /// <response code="200">Successfully removed a relationship.</response>
+        /// <response code="400">Invalid DeletePersonalRelationshipRequest received</response>
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        [HttpDelete]
+        [Route("relationships/personal/{id:long}")]
+        public IActionResult RemovePersonalRelationshipRequest(long id)
```

Comment:
```suggestion
        public IActionResult RemovePersonalRelationship(long id)
```

Reply:
Could we add controller tests for this endpoint? See https://github.com/LBHackney-IT/social-care-case-viewer-api/tree/development/SocialCareCaseViewerApi.Tests/V1/Controllers/Relationship for examples.